platforms/software/dependency-management/src/main/java/org/gradle/internal/component/external/model/DefaultImmutableCapability.java

```diff
@@ -28,6 +29,10 @@ public static DefaultImmutableCapability defaultCapabilityForComponent(ModuleVer
         return new DefaultImmutableCapability(identifier.getGroup(), identifier.getName(), identifier.getVersion());
     }
 
+    public static DefaultImmutableCapability defaultCapabilityForComponent(ProjectComponentIdentifier identifier) {
```

Comment:
This is not correct. You cannot derive the default capability of a project solely on its `ProjectComponentIdentifier`.

Reply:
This wasn't used, just experimental code I missed deleting.  Deleting.